src/content/plugins/commons-chunk-plugin.md

```diff
@@ -6,9 +6,14 @@ contributors:
   - christopher4lis
   - kevinzwhuang
   - jdbevan
+  - jeremenichelli
 ---
 
-The `CommonsChunkPlugin` is an opt-in feature that creates a separate file (known as a chunk), consisting of common modules shared between multiple entry points. By separating common modules from bundles, the resulting chunked file can be loaded once initially, and stored in cache for later use. This results in pagespeed optimizations as the browser can quickly serve the shared code from cache, rather than being forced to load a larger bundle whenever a new page is visited.
+The `CommonsChunkPlugin` is an opt-in feature that creates a separate file (known as a chunk), consisting of common modules shared between multiple entry points.
+
+W> CommonsChunkPlugin has been deprecated in webpack v4 legato. If you want to know how chunks are treated in the newest version checkout [SplitChunksPlugin](/plugins/split-chunks-plugin/).
+
+By separating common modules from bundles, the resulting chunked file can be loaded once initially, and stored in cache for later use. This results in pagespeed optimizations as the browser can quickly serve the shared code from cache, rather than being forced to load a larger bundle whenever a new page is visited.
 
 ```javascript
 new webpack.optimize.CommonsChunkPlugin(options)
```

Comment:
shouldn't this be removed? or replaced with:
```json
optimization: {
  splitChunks: {
    name: 'all',
  },
}
```

Reply:
This is the old CommonsChunkPlugin page. We are keeping the old documentaiton for teams that are already using it and haven't migrated to v4. We are adding a warning on top talking about the deprecation of it from now on.